metadata/latest/Azure.Batch.FileStaging.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
For these newly added files do they need more properties?

Reply:
```
azurebatchfilestaging,Azure.Batch.FileStaging
```
This is the only place we defined the package in docs metadata and `azure-dotnet.csv`. I didn't find any other info to fill in.